### PR TITLE
Update CUDA toolkit for Jetson build to 11.8

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- updates CUDA toolkit for Jetson build to 11.8

### Affected modules and functionalities:
- docker/Dockerfile.build.aarch64-linux

### Tests:

- [x] Existing tests apply
  - TL0_python-self-test_xavier
  - TL0_python-self-test_xavier
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [ ] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

File: docker/Dockerfile.build.aarch64-linux [hunk 1]
```diff
@@ -1,8 +1,9 @@
-ARG AARCH64_BASE_IMAGE=nvidia/cuda:11.4.2-devel-ubuntu20.04
+ARG AARCH64_BASE_IMAGE=nvidia/cuda:11.8.0-devel-ubuntu20.04
 FROM ${AARCH64_BASE_IMAGE}
 
 ENV DEBIAN_FRONTEND=noninteractive
-ENV CUDA_CROSS_VERSION=11-4
+ENV CUDA_CROSS_VERSION=11-8
+ENV CUDA_CROSS_VERSION_DOT=11.8
```
Comment (JanuszL) [inline docker/Dockerfile.build.aarch64-linux:6]: Docker bash doesn't support substitution.

File: tools/stub_generator/npp.json [hunk 2]
```diff
@@ -14,6 +14,16 @@
          "return_type":"NppLibraryVersion *",
          "not_found_error":"nullptr" },
       "nppiColorTwist32f_8u_C3R": {},
-      "nppiColorTwist32f_8u_C1R": {}
+      "nppiColorTwist32f_8u_C1R": {},
+            "nppiRemap_64f_C1R_Ctx": {},
+      "nppiRemap_64f_C3R_Ctx": {},
```
Comment (szalpal) [inline tools/stub_generator/npp.json:19]:
```suggestion
      "nppiColorTwist32f_8u_C1R": {},
      "nppiRemap_64f_C1R_Ctx": {},
      "nppiRemap_64f_C3R_Ctx": {},
```
The indentation looks off

**Review by szalpal — APPROVED** [review]

**Comment by szalpal** [comment]
Should the `cmake/Dependencies.aarch64-qnx.cmake` file be updated too?

File: tools/stub_generator/npp.json (hunk 2 above)
Comment (JanuszL) [reply tools/stub_generator/npp.json]: Fixed

**Comment by JanuszL** [comment]
> Should the `cmake/Dependencies.aarch64-qnx.cmake` file be updated too with the `nppig` linking routines?

We no longer build nor support this configuration anymore.

**Review by awolant — APPROVED** [review]

**Pull request merged**